examples/embedding_n_remote/README.txt

```diff
@@ -0,0 +1,4 @@
+Embedding and Remote Session Example
+====================================
+
+Below is the same example first demonstrated using an embedding instance, and next demonstrated using a remote session.
```

Comment:
```suggestion
Here is the same example, first demonstrated using an embedding instance and then demonstrated using a remote session.
```
Google dev doc style guide says not to use above, below rather preceding or following. Here, I don't think a locator is needed.